### ISSUE TITLE
is there a --quiet option

### ISSUE THREAD

**Issue by wayneseymour** [jscs-dev/node-jscs#425]
For my `commit hook`, I'd prefer `jscs` to say nothing if all is well.
Does this already exist?

**Comment by mosbth** [comment]
@mikesherov, still, it would be handy to have a `--quiet` option that is completely silent upon success. CSSLint is another tool that has such option.

I did not want to use the `-r inline` option, I'd prefer the more verbose report mode as default is. So that did not do the trick for me.

I tried the `grep` variant like this:
```
jscs  code.js | grep -v "No code style errors found."
```

My problem was that the exit-code was from `grep` and not from `jscs`. So I got a exit code of 1 from `grep` while I expected an exitcode  of 0 from jscs. 

I sorted it out using PIPESTATUS (bash) as below shows. **It works for me so I can use it like that.** 
```
jscs  code.js | grep -v "No code style errors found." ; echo ${PIPESTATUS[0]}
```

But a `--quiet` option would perhaps be a more solid solution.
